library/vyos_l3_interfaces.py

```diff
@@ -122,249 +123,745 @@
+          - address:203.0.113.65/26
         vifs:
-          - vlan_id: 101
+          - vlan_id:101
             ipv4:
-              - address: 192.0.2.71/28
-              - address: 198.51.100.131/25
-          - vlan_id: 102
+              - address:192.0.2.71/28
+              - address:198.51.100.131/25
+          - vlan_id:102
             ipv6:
-              - address: 2001:db8:1000::5/38
-              - address: 2001:db8:1400::3/38
-    state: merged
-
+              - address:2001:db8:1000::5/38
+      - name:bond0
+        ipv4:
+           - address:172.0.0.11/26
+        ipv6:
+           - address:2001:db8:1000::5/38
+           - address:2001:db8:1400::3/38
+    state:merged
+#
+#
+# ------------------------
+# Module Execution Results
+# ------------------------
+#
+# "before":[
+#        {
+#            "name":"eth3"
+#       },
+#        {
+#            "name":"eth2"
+#        },
+#        {
+#            "ipv4":[
+#                {
+#                    "address":"dhcp"
+#                }
+#            ],
+#            "name":"eth0"
+#        }
+#    ]
+#
+# "commands":[
+#        "set interfaces ethernet eth2 address '2001:db8:100::2/32'",
+#        "set interfaces ethernet eth2 address '2001:db8:400::10/32'",
+#        "set interfaces ethernet eth2 address '192.0.2.10/28'",
+#        "set interfaces ethernet eth2 address '198.51.100.40/27'",
+#        "set interfaces ethernet eth3 address '203.0.113.65/26'",
+#        "set interfaces ethernet eth3 vif 101 address '192.0.2.71/28'",
+#        "set interfaces ethernet eth3 vif 101 address '198.51.100.131/25'",
+#        "set interfaces ethernet eth3 vif 102 address '2001:db8:1000::5/38'",
+#        "set interfaces ethernet eth3 vif 102 address '2001:db8:1400::3/38'",
+#        "set interfaces bonding bond0 address '172.0.0.11/26'",
+#        "set interfaces bonding bond0 address '2001:db8:1000::5/38'"
+#    ]
+# "after":[
+#        {
+#            "ipv4":[
+#                {
+#                    "address":"203.0.113.65/26"
+#                }
+#            ],
+#            "name":"eth3",
+#            "vifs":[
+#                {
+#                    "ipv6":[
+#                        {
+#                            "address":"2001:db8:1000::5/38"
+#                        },
+#                        {
+#                            "address":"2001:db8:1400::3/38"
+#                        }
+#                    ],
+#                    "vlan_id":"102"
+#                },
+#                {
+#                    "ipv4":[
+#                        {
+#                            "address":"192.0.2.71/28"
+#                        },
+#                        {
+#                            "address":"198.51.100.131/25"
+#                       }
+#                    ],
+#                    "vlan_id":"101"
+#                }
+#            ]
+#        },
+#        {
+#            "ipv4":[
+#                {
+#                    "address":"172.0.0.11/26"
```

Comment:
Please use IP addresses valid for documentation. Refer: https://tools.ietf.org/html/rfc5737